PurchasesTests/Purchasing/RCIntroEligibilityTests.swift

```diff
@@ -11,24 +11,28 @@ import Nimble
 import Purchases
 @testable import PurchasesCoreSwift
 
-class RCIntroEligibilityTests: XCTestCase {
+class IntroEligibilityTests: XCTestCase {
     func testInitWithEligibilityStatusCodeUnknown() {
-        let introElegibility = RCIntroEligibility(eligibilityStatusCode: NSNumber(value: IntroEligibilityStatus.unknown.rawValue))
-        expect(introElegibility?.status) == RCIntroEligibilityStatus.unknown
+        let introElegibility = try! IntroEligibility(eligibilityStatusCode: NSNumber(value: IntroEligibilityStatus.unknown.rawValue))
+        expect(introElegibility.status) == IntroEligibilityStatus.unknown
     }
     
     func testInitWithEligibilityStatusCodeIneligible() {
-        let introElegibility = RCIntroEligibility(eligibilityStatusCode: NSNumber(value: IntroEligibilityStatus.ineligible.rawValue))
-        expect(introElegibility?.status) == RCIntroEligibilityStatus.ineligible
+        let introElegibility = try! IntroEligibility(eligibilityStatusCode: NSNumber(value: IntroEligibilityStatus.ineligible.rawValue))
+        expect(introElegibility.status) == IntroEligibilityStatus.ineligible
     }
 
     func testInitWithEligibilityStatusCodeEligible() {
-        let introElegibility = RCIntroEligibility(eligibilityStatusCode: NSNumber(value: IntroEligibilityStatus.eligible.rawValue))
-        expect(introElegibility?.status) == RCIntroEligibilityStatus.eligible
+        let introElegibility = try! IntroEligibility(eligibilityStatusCode: NSNumber(value: IntroEligibilityStatus.eligible.rawValue))
+        expect(introElegibility.status) == IntroEligibilityStatus.eligible
```

Comment:
`RCIntroEligibility` was the obj-c counterpart of `IntroEligibilityStatus`, and was added because obj-c enums aren't compatible with swift enums, so this translation no longer makes sense - we should just unify the code for both, by removing the old `IntroEligibilityStatus` (since that one wasn't public) and replacing all of its uses with `IntroEligibility`